server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -129,13 +130,15 @@ export default class ActivitiesMigrationController {
         return null
       }),
 
-      this.nomisMigrationService.findActivitiesSuspendedAllocations(filter, context(res)).catch(error => {
-        errors.push({
-          text: `Failed to find suspended allocations due to error: ${error.message}`,
-          href: '',
-        })
-        return []
-      }),
+      this.nomisPrisonerService
+        .findActivitiesSuspendedAllocations(filter, await activityCategories, context(res))
+        .catch(error => {
+          errors.push({
+            text: `Failed to find suspended allocations due to error: ${error.message}`,
+            href: '',
+          })
+          return []
+        }),
```

Comment:
The main change is not to proxy this request via the migration API so it can get the DPS activity categories - we get them in this service now so call the NOMIS API directly. This will reduce a fair bit of traffic once all checks are in.